wiki/Offset/Local_offset/en.md

```diff
@@ -1,19 +1,28 @@
 ---
-stub: true
+tags:
+  - offline offset
 ---
 
 # Local song offset
 
 *For other uses, see [Offset](/wiki/Offset).*
 
-**Local song offset** is an offset which moves the appearance of [hit objects](/wiki/Hit_object) in relation to the audio of individual [beatmaps](/wiki/Beatmap). This can be helpful for players who experience auditory or visual delays. Local song offset works alongside the [global offset](/wiki/Client/Options/Universal_offset) to calculate a total offset.
+**Local** (or *offline*) **song offset** is a setting which moves the appearance of [hit objects](/wiki/Hit_object) in relation to the audio of individual [beatmaps](/wiki/Beatmap). This can be helpful for players who experience auditory or visual delays. Local song offset works alongside the [global offset](/wiki/Client/Options/Universal_offset) to calculate a total offset.
+
+## Behaviour
+
+Local offset is customized on a per-beatmap basis. It works by moving all hit objects and [timing points](/wiki/Client/Beatmap_editor/Timing) of a beatmap by a specificed amount of milliseconds. Negative values move objects earlier in time (closer to the beginning), while positive values move them later, which is notably different from how [universal offset](/wiki/Client/Options/Universal_offset) functions.
```

Comment:
instead of the first two you could just say "hit objects" and be done with it, since it implies every aspect of them. as for video/sb, I simply forgot.

> i'm not sure what moving "timing points" would imply here

that they are also affected by the local offset: if you dial it up to 100, custom sliders velocities won't suddently stop applying to the first hit objects of every section with velocity change

Reply:
(I have changed the wording to cover all possible gameplay and beatmap elements affected)